plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation_test.go

```diff
@@ -37,12 +37,15 @@ func TestBalancedResourceAllocation(t *testing.T) {
 	}
 	machine1Spec := v1.PodSpec{
 		NodeName: "machine1",
+		Affinity: &v1.Affinity{},
```

Comment:
Why is this not omit_if_empty?  It seems like we're filling in blanks in a lot of locations. 

We don't do that with other fields.